arrayOperationTask/script.js

```diff
@@ -1,33 +1,16 @@
 // reverse
-const reverseFun = (data)=>{
-   let reverserArr = []
-   let datalength= data.length
-   for(let i=0;i<datalength;i++){
-      let poppedItem = data.pop()
-      reverserArr.push(poppedItem)
-   }
-   return reverserArr
-}
+const reverseFun = (data) => {
+	return data.map((item,index,array)=>array[array.length-1-index])

```

Comment:
@eligifty-coder, this is a really good attempt! Very impressive!
Again, we could have shortened this to:

```
const reverseFun = (data) => data.map((item, index, array) => array[array.length - 1 - index])
```
However, this still uses a new array `array` (the 3rd argument to the map function) to store the results of the reverse operation.

The ideal solution will swap the entries in the original array. No need to block this PR, we will talk more about this during our tech interview prep sessions.